README.md

```diff
@@ -146,19 +146,15 @@ Key:
 See the [CHANGELOG](./CHANGELOG.md) for a detailed description of changes
 between java-client versions.
 
-## Community, Support, Discussion
+## Contributing
 
-You can reach the maintainers of this project at [SIG API Machinery](https://github.com/kubernetes/community/tree/master/sig-api-machinery). If you have any problem with the package or any suggestions, please file an [issue](https://github.com/kubernetes-client/java/issues).
-
-### Code of Conduct
```

Comment:
I feel like the code of conduct is important enough it should be on the front page, please leave this here too.

Reply:
Done.